awx/ui_next/src/components/PaginatedDataList/PaginatedDataList.jsx

```diff
@@ -82,10 +81,8 @@ class PaginatedDataList extends React.Component {
         ];
     const queryParams = parseQueryString(qsConfig, location.search);
 
-    const itemDisplayName = ucFirst(pluralize(itemName));
-    const itemDisplayNamePlural = ucFirst(
-      itemNamePlural || pluralize(itemName)
-    );
+    const itemDisplayName = ucFirst(itemName);
+    const itemDisplayNamePlural = ucFirst(itemName);
```

Comment:
@keithjgrant perhaps I don't understand what you are saying but `itemName` is a word(s) in singular or plural form that is passed in from the component using PDL. That component determines the plurality of `itemName` so I don't quite understand why we need to pass both singular and plural words.  I do think, though, that I can get ride of both `itemDisplayName` and `itemDisplayNamePlural` as well as `ucFirst()`

Reply:
my point here is you have two variables with the same value. so get rid of the second one and just use the first one throughout... In fact, you can get rid of both `itemDisplayName` and `itemDisplayNamePlural`, and just use `itemName` throughout, since it's already capitalized by the parent components